python/ray/runtime_context.py

```diff
@@ -364,9 +364,7 @@ def get_runtime_context():
 
         >>> import ray
         >>> # Get the job id.
-        >>> ray.get_runtime_context().job_id # doctest: +SKIP
-        >>> # Get all the metadata.
-        >>> ray.get_runtime_context().get() # doctest: +SKIP
+        >>> ray.get_runtime_context().get_job_id() # doctest: +SKIP
```

Comment:
Why is it skipped, would it just work if you remove the `SKIP`?

Reply:
fixed.